src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -226,8 +226,9 @@ def load_arguments(self, _):
             c.argument('settings', nargs='+', help="space-separated app settings in a format of `<name>=<value>`")
             c.argument('setting_names', nargs='+', help="space-separated app setting names")
         with self.argument_context(scope + ' config ssl import') as c:
+            c.argument('certificate_name', help='The name of the certificate to be imported')
+            c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault', deprecate_info=c.deprecate(expiration='2.15.0'))
```

Comment:
Can we update the help to include to use the new argument moving forward instead of the key_vault_certificate_name? @qwordy can you tell us the recommended approach here?

Reply:
https://github.com/Azure/azure-cli/blob/a990e9626392e1f2bf8da0e8afefd24c8a518f48/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments
Use `redirect=` redirect users to the new argument

Reply:
@singlessis please follow the documentation for this breaking change.